Linear/CMakeLists.txt

```diff
@@ -27,6 +28,7 @@ if(NOT TARGET Beam)
 endif()
 
 file(GLOB LinEl_SRCS *.C)
+list(APPEND LinEl_SRCS ../Shell/SIMKLShell ../Shell/KirchhoffLoveShell)
```

Comment:
missing .C ?

Reply:
Hmm, looks indeed so. But it actually worked even though. Will fix anyway.